app/spec/services/provider_search_service_spec.rb

```diff
@@ -19,12 +21,96 @@
     end
 
     it "returns results with correct structure" do
+      stub_const("ProviderSearchService::SUPPORTED_PROVIDERS", [ :pinwheel ])
       results = service.search("test")
       result = results.first
+      puts result
 
       expect(result).to have_attributes(
         provider_name: :pinwheel,
-        provider_options: a_hash_including(:response_type, :provider_id),
+        provider_options: an_object_having_attributes(response_type: a_kind_of(String), provider_id: a_kind_of(String)),
```

Comment:
```suggestion
        provider_options: have_attributes(response_type: a_kind_of(String), provider_id: a_kind_of(String)),
```

(For consistency. It's a synonym.)

Reply:
updated.